Makefile

```diff
@@ -31,22 +30,24 @@ build: | $(O) ## Build reflect binaries
 install: ## Build and install binaries in $GOBIN
 	go install -ldflags='$(GO_LDFLAGS)' $(CMDS)
 
-tiny: | $(O) ## Build and test for tinygo
-	tinygo test ./...
+tiny: | $(O) ## Build for tinygo / wasm
 	# Optimise tinygo output for size, see https://www.fermyon.com/blog/optimizing-tinygo-wasm
```

Comment:
I noticed when running `make` that this comment line gets printed. Perhaps move it to before the target definition so it doesn't get printed? not ideal, but is that better?

Reply:
Done.